reactjs-films-homework/src/components/ErrorBoundary/ErrorBoundary.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { Component, ErrorInfo, ReactNode } from 'react';
+import styles from './ErrorBoundary.module.scss';
+
+interface Props {
```

Comment:
You can use 'I' prefix for interfaces for properly identifying interfaces in the code visually. This name could be 'IErrorBounaryProps'

Reply:
I'm not a little attentive